09-redux/classworks/Volodkova_Anna/src/App.js

```diff
@@ -0,0 +1,77 @@
+import React, {useEffect, useState, useRef} from 'react';
+import {useRouteMatch} from "react-router-dom";
+import {useSelector, useDispatch} from 'react-redux';
+import Header from 'components/header'
+import Pages from 'pages';
+import {withRedditApi} from './hooks/useRedditApi';
+import {setPostsNull, setSearchValue, fetchPosts} from "./resourses/actions";
+
+function App(props) {
+  const dispatch = useDispatch();
+  const getSearchValue = useSelector((state) => state.value);
```

Comment:
selectors should be places in separate file like `searchValue.selectors.js`. also every little part of store should be an object (e.g. `store = { search: { value: 'Something' }, users: { list: [] }, anotherPart: {} }`)